cloudwatch_logs_common/src/utils/cloudwatch_facade.cpp

```diff
@@ -173,22 +206,25 @@ Aws::CloudWatchLogs::ROSCloudWatchLogsErrors CloudWatchFacade::CreateLogStream(
   return status;
 }
 
-Aws::CloudWatchLogs::ROSCloudWatchLogsErrors CloudWatchFacade::GetLogStreamToken(
-  const std::string & log_group, const std::string & log_stream, Aws::String & next_token)
+Aws::CloudWatchLogs::ROSCloudWatchLogsErrors CloudWatchFacade::CheckLogStreamExist(
+  const std::string & log_group,
+  const std::string & log_stream,
+  Aws::CloudWatchLogs::Model::LogStream * log_stream_object)
 {
   Aws::CloudWatchLogs::ROSCloudWatchLogsErrors status = CW_LOGS_SUCCEEDED;
 
   Aws::CloudWatchLogs::Model::DescribeLogStreamsRequest describe_log_stream_request;
   describe_log_stream_request.SetLogGroupName(log_group.c_str());
   describe_log_stream_request.SetLogStreamNamePrefix(log_stream.c_str());
-  ;
 
   const auto & response = this->cw_client_.DescribeLogStreams(describe_log_stream_request);
   if (!response.IsSuccess()) {
```

Comment:
Same as above: if there is a request failure then we don't know whether the log stream exists or not